src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let totalCost = days * 40;
```

Comment:
Prefer `const` over `let` for `totalCost` since its value is not being reassigned after the initial calculation, which aligns with checklist item 3.